webapp/config/packages/fos_rest.yaml

```diff
@@ -16,3 +16,4 @@ fos_rest:
             Symfony\Component\HttpKernel\Exception\NotAcceptableHttpException: true
             Symfony\Component\HttpKernel\Exception\NotFoundHttpException: true
             Symfony\Component\HttpKernel\Exception\UnauthorizedHttpException: true
+    body_listener: ~
```

Comment:
are you kidding me? Is that everything? :-D

Reply:
![](http://www.reactiongifs.com/r/mgc.gif)